### PR TITLE
[INLONG-8389][Manager] Fix the tenant interception failure when authentication is disable

### LINKED ISSUE #8389: [Bug][Manager] Tenant interception failure when authentication is disable
### What happened

Tenant interception failure when authentication is disable because there is no any login user info.

### What you expected to happen

Use the default user "admin" and default tenant "public" if authentication is disable and request doesn't specify anyone.

### How to reproduce

Disable the request authentication and request any interface.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #8389 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/auth/tenant/TenantAuthenticatingFilter.java [hunk 1]
```diff
@@ -48,23 +55,24 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
...
         // pre-check
         if (!subject.isAuthenticated()) {
-            log.error("The request should be authed before tenant authentication, user:{}, path:{}",
-                    subject.getPrincipal(),
-                    httpServletRequest.getServletPath());
-            ((HttpServletResponse) response).sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
-            return;
+            log.debug("the request is not authed before tenant authentication, use default user:{}, path:{}",
+                    DEFAULT_USER, httpServletRequest.getServletPath());
+            loginUserInfo = defaultUserInfo();
```
Comment (healchow) [inline inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/auth/tenant/TenantAuthenticatingFilter.java:63]: Will it exist that when authentication is enabled, the user is not logged, and finally the default user info is set here by mistake?

Comment (fuweng11) [reply inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/auth/tenant/TenantAuthenticatingFilter.java:63]: I think it's only necessary to set a default user when OpenAPI authentication is not enabled.

Comment (vernedeng) [reply inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/auth/tenant/TenantAuthenticatingFilter.java]: The idea of @fuweng11 is a good solution.

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptor.java [hunk 2]
```diff
@@ -171,12 +171,13 @@ private boolean isPrimitiveOrWrapper(Object obj) {
     private String getTenant() {
         UserInfo userInfo = LoginUserUtils.getLoginUser();
         if (userInfo == null) {
-            throw new BusinessException("Current user is null, please login first");
```
Comment (fuweng11) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/interceptor/MultiTenantInterceptor.java:174]: An exception should be thrown if the `userInfo ` is empty

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/auth/openapi/OpenAPIFilter.java [hunk 3]
```diff
@@ -89,32 +89,32 @@ private SecretToken parseBasicAuth(HttpServletRequest servletRequest) {
         String basicAuth = servletRequest.getHeader(BasicAuth.BASIC_AUTH_HEADER);
         if (StringUtils.isBlank(basicAuth)) {
             log.error("basic auth header is empty");
-            return null;
+            return new SecretToken();
```
Comment (fuweng11) [inline inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/auth/openapi/OpenAPIFilter.java:92]: Add some comment to explain it.

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/auth/openapi/OpenAPIFilter.java [hunk 4]
```diff
@@ -85,36 +85,37 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         }
     }
 
+    // return empty token if parse failed.
     private SecretToken parseBasicAuth(HttpServletRequest servletRequest) {
         String basicAuth = servletRequest.getHeader(BasicAuth.BASIC_AUTH_HEADER);
         if (StringUtils.isBlank(basicAuth)) {
             log.error("basic auth header is empty");
```
Comment (fuweng11) [inline inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/auth/openapi/OpenAPIFilter.java:92]: log.warn

**Review by fuweng11 — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Pull request merged**